apps/web/src/features/myAccounts/components/AccountItems/MultiAccountItem.tsx

```diff
@@ -127,6 +129,24 @@ function useMultiAccountItemData(multiSafeAccountItem: MultiChainSafeItem) {
 
   const deployedChainIds = useMemo(() => sortedSafes.map((safe) => safe.chainId), [sortedSafes])
 
+  const hasSafenetFeature = useHasSafenetFeature()
+  const { data: safenetConfig } = useGetSafenetAccountQuery({ safeAddress: address }, { skip: !hasSafenetFeature })
```

Comment:
This request is sent for every account added to the sidebar. The succesful responses are handled by RTK and cached, but the failed ones are not.

This means that this request is sent every time the sidebar is opened for all the non-Safenet accounts, creating a possible issue if there are many Safes added.

We should check if it makes sense to avoid returning an Error 404 when Safes are not found in the service so the response is cached as well.

Reply:
Agree! I think it makes sense to just return an empty response with 200 OK.